other/housing/housing.jl

```diff
@@ -16,30 +36,60 @@ isfile("housing.data") ||
 
 rawdata = readdlm("housing.data")'
 
-# The last feature is our target -- the price of the house.
-split_ratio = 0.1 # For the train test split
+#-
+
+# Specifying the split ratio and **x** and **y**
+split_ratio = 0.1 
 
 x = rawdata[1:13,:] |> gpu
 y = rawdata[14:14,:] |> gpu
 
-# Normalise the data
+# ### Normalising
+# What is the need ? <br>
```

Comment:
Why is there a `<br.` tag here?

Reply:
Sorry. I thought since this would be interpreted as Markdown, I might as well add a `br` to break a line. Thanks for pointing it out.

Reply:
e317015 : Removed the `<br>` tag.